src/Shared/PackageManagers/BaseProjectManager.cs

```diff
@@ -230,13 +231,14 @@ public static async Task<JsonDocument> GetJsonCache(HttpClient client, string ur
             Logger.Trace("Loading Uri...");
             HttpResponseMessage result = await client.GetAsync(uri);
             result.EnsureSuccessStatusCode(); // Don't cache error codes.
-            long contentLength = result.Content.Headers.ContentLength ?? 8192;
+            long contentLength = 0;
```

Comment:
Nit: It's actually slightly better not to assign a default value here since we expect any/all code paths below to set the value. And the compiler will in fact help us make sure that all code paths set the value.

```suggestion
            long contentLength;
```